floodviz/templates/sitemap.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link type="text/css" href="{{ url_for('static', filename='bower_components/bootstrap/dist/css/bootstrap.css') }}"/>
+    <title>Site Map</title>
+</head>
+<body>
```

Comment:
It would be better to have a div specifically for the map. I am fine with this for now as this is more of a proof of concept.